### DISCUSSION TITLE
有人要jetbrains licence吗

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#211] [General]
我 ~不小心~ 提交了申请， 但是我已经有licence了
<img width="1341" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/43a5ca1b-3d8c-461a-b75e-a083c9e36836">
<img width="552" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/d2e94de8-f8cc-4c73-a861-5d9138bc1a48">

**Comment by langningchen** [comment]
emm
